src/main/java/org/broadinstitute/hellbender/engine/ReferenceDataSource.java

```diff
@@ -75,9 +47,7 @@ public ReferenceDataSource( final File fastaFile ) {
      * @param interval query interval
      * @return a ReferenceSequence containing all bases spanning the query interval, prefetched
      */
-    public ReferenceSequence queryAndPrefetch( final SimpleInterval interval ) {
-        return queryAndPrefetch(interval.getContig(), interval.getStart(), interval.getEnd());
-    }
+    public ReferenceSequence queryAndPrefetch( final SimpleInterval interval );
```

Comment:
Add a default implementation for this method in the interface that just calls into `queryAndPrefetch( final String contig, final long start , final long stop)`